packages/gatsby-plugin-feed/src/serialize.js

```diff
@@ -0,0 +1,16 @@
+export default function serialize({
```

Comment:
I'm inclined to think that even though the query format is standardized, we should still include a way for users to customize feed options via a callback.